python/ray/tests/test_ray_get.py

```diff
@@ -0,0 +1,59 @@
+import os
+import sys
+import threading
+import time
+
+import numpy as np
+import pytest
+
+import ray
+
+
+@pytest.fixture(autouse=True)
+def cap_object_transfer_throughput():
+    # This will make the object transfer slower and allow the test to
+    # interleave Get requests.
+    # TODO(57923): Make this not autouse when a second test is added to this file.
+    os.environ["RAY_object_manager_max_bytes_in_flight"] = str(1024**2)
+
+
+def test_multithreaded_ray_get(ray_start_cluster):
+    # This test tries to get a large object from the head node to the worker node
+    # while making many concurrent ray.get requests for a local object in plasma.
+    # TODO(57923): Make this not rely on timing if possible.
+    ray_cluster = ray_start_cluster
+    ray_cluster.add_node()
+    ray.init()
```

Comment:
```suggestion
    ray.init(address=ray_cluster.address)
```
not strictly required, but good practice